### PR TITLE
Give an Option to always show interstate edge labels

### PR DESCRIPTION
**Pull request by phschaad**

### CODE REVIEW

File: src/renderer/renderer_elements.ts [hunk 1]
```diff
@@ -854,6 +857,30 @@ export class InterstateEdge extends Edge {
             container.style.display = 'none';
     }
 
+    public drawLabel(
```
Comment (tbennun) [inline src/renderer/renderer_elements.ts:860]: This doesn't make space for the rest of the graph. Wouldn't this overlap other nodes? What does this look like?

**Review by tbennun — COMMENTED** [review]
Open questions but otherwise good.

File: src/renderer/renderer_elements.ts (hunk 1 above)
Comment (phschaad) [reply src/renderer/renderer_elements.ts:860]: It _can_ in theory overlap with other nodes, as yes, there is no space reservation. However, the placement is at the start of the interstate edge (see image). So the overlap should only happen if:
a) the layout is bad to begin with (nodes basically touching), or
b) the interstate edge label is extremely long (e.g., very complex, huge condition)

image

Comment (tbennun) [reply src/renderer/renderer_elements.ts:860]: Ok. Thanks for the screenshot

**Review by tbennun — APPROVED** [review]

**Pull request merged**